src/validations/workoutForm.js

```diff
@@ -0,0 +1,36 @@
+import moment from 'moment';
+import _ from 'lodash';
+
+import { handleInvalidInput, emptyInputs } from './form';
+import { filterByDate, isAfterDate } from '../queries/date';
+import { totalWorkoutsDuration } from '../queries/workoutLog';
+
+export const validateForm = (component, form) => {
+  let valid = true;
```

Comment:
Change this logic to use a single var called `errors` - which is an object that holds all the element props plus their possible errors - and at the end of the method, check if it is empty or not. If so,  call the handleError passing the errors var and return true, otherwise, return false.